client/landing/subscriptions/components/site-subscriptions-manager/site-row.tsx

```diff
@@ -101,15 +112,38 @@ const SiteRow = ( {
 		SubscriptionManager.useSiteUnsubscribeMutation();
 	const { mutate: resubscribe } = SubscriptionManager.useSiteSubscribeMutation();
 
+	// Tracks events recording
+	const recordSiteIconClicked = useRecordSiteIconClicked();
+	const recordSiteTitleClicked = useRecordSiteTitleClicked();
+	const recordSiteUrlClicked = useRecordSiteUrlClicked();
+	const recordNotificationsToggle = useRecordNotificationsToggle();
+	const recordPostEmailsToggle = useRecordPostEmailsToggle();
+	const recordCommentEmailsToggle = useRecordCommentEmailsToggle();
+	const recordPostEmailsSetFrequency = useRecordPostEmailsSetFrequency();
+	const recordSiteUnsubscribed = useRecordSiteUnsubscribed();
+	const recordSiteResubscribed = useRecordSiteResubscribed();
+
+	// Make object assignment a little easier
+	const SOURCE_SUBSCRIPTIONS_SITE_LIST = 'subscriptions-site-list';
```

Comment:
It is a good practice to define such constants outside the component's render function.